### DISCUSSION TITLE
Buyers need to buy things

### DISCUSSION THREAD

**Discussion by eprocurementontology** [eprocurementontology/eprocurementontology#487] [Ideas]
Jáchym Hercher submitted the following use case to be further developed:
Buyers need to buy things, which means following the e-procurement phases described in the Project Charter (e.g. publishing notices, sending information about who won a contract, signing contracts, drafting and archiving individual procurement reports in the sense of 2014/24/EU Art. 84 etc. etc.). This includes:

1. Creating new information (e.g. description of the procurement, giving points for award criteria).
2. Reusing information from different databases and domains, such as
         - business registries (to reduce administrative burden and ensure consistency of information);
         - tax, social payments, etc. systems (to verify that potential contractors meet selection criteria).
3. (Sending information to other systems to ensure transparency etc. requirements are met, e.g. contract registers).

**Comment by muricna** [comment]
I am not sure what is meant by creating new information?

**Comment by JachymHercher** [comment]
For example, when the buyer writes "We want to buy a vehicle which can clean up snow 20 times per year in our village, it needs to be able to use green fuel", then this text is newly created information. (Which then gets sent to various documents to be transmitted to Economic Operators, archived, etc.)

**Comment by muricna** [comment]
New compared to what?

**Comment by JachymHercher** [comment]
Ah, I see the ambiguity. I mean new as in not existing before (not new as in updated).
